antea/scripts/characterize_coincidences.py

```diff
@@ -10,7 +10,6 @@
 import antea.elec.shaping_functions as shf
 import antea.mcsim.sensor_functions as snsf
 
-from antea.io.mc_io import read_sensor_bin_width_from_conf
```

Comment:
Yes, because it's useful to analyze the simulations produced so far and it's harmless to keep it (one would simply not use it with the new format). On the other hand, I changed the function `find_coincidence_timestamps` to avoid having two different functions with a similar name; in that case, the user should write their own function instead.

Reply:
What do you mean with "to avoid having two different functions with a similar name"?

Reply:
I didn't want to have two functions that do the same, one with the + sign and the other one with the - sign and call them `blablabla_old` or `blablabla_new`.

Reply:
Ah right, I agree!